tests/unit/admin/views/test_projects.py

```diff
@@ -99,6 +99,7 @@ def test_gets_project(self, db_request):
             "project": project,
             "releases": [],
             "maintainers": roles,
+            "package_roles": [],
```

Comment:
I'm not sure if we want to create some representations by using the `RoleFactory`. In this case what should be the difference between `maintainers` and `package_roles`?